script/parseTs.ts

```
@@ -0,0 +1,152 @@
+                */
+
+                break;
+            case ts.SyntaxKind.ClassDeclaration:
+                const classDeclaration = node as ts.ClassDeclaration;
+                // console.log(`Class: ${classDeclaration.name?.getText().replaceAll('_','.')}`);
+                const pyClass = convertTsClassToPython(classDeclaration.getText());
+                    // .replace('_','.')
+                functions.push(pyClass);
+                break;
+            case ts.SyntaxKind.TypeAliasDeclaration:
+                const alias = node;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)